src/view/dataElements/conditionalValue.jsx

```diff
@@ -10,76 +10,111 @@ governing permissions and limitations under the License.
 */
 
 import React from 'react';
-import { TextField, Flex } from '@adobe/react-spectrum';
+import { TextField, Flex, Checkbox, View } from '@adobe/react-spectrum';
+import { connect } from 'react-redux';
+import { formValueSelector } from 'redux-form';
 import ValueComparison, {
   formConfig as valueComparisonFormConfig
 } from '../conditions/valueComparison';
 import WrappedField from '../components/wrappedField';
+import InfoTip from '../components/infoTip';
 import mergeFormConfigs from '../utils/mergeFormConfigs';
+import { isNumberLike } from '../utils/validators';
 
-const ConditionalField = () => {
+const ReturnField = ({
+  shouldReturnValue,
+  checkboxName,
+  checkboxLabel,
+  inputName,
+  inputLabel,
+  inputPlaceholder
+}) => {
   return (
     <>
-      <WrappedField
-        label="If true, return this string value"
-        name="conditionalValue"
-        placeholder="Value if true"
-        width="size-3000"
-        component={TextField}
-        supportDataElement
-        isRequired
-      />
+      <Flex alignItems="center">
+        <WrappedField name={checkboxName} component={Checkbox}>
+          {checkboxLabel}
+        </WrappedField>
+        <View position="relative" left="-1rem">
+          <InfoTip>
```

Comment:
These tooltip icons look super big. Are they really this big throughout our extension? If these are the first, let's set a precedent to make them smaller. FWIW, I believe the Lens UI is `5px`.

Reply:
The InfoTip component is not a component introduced by this PR.